New PR, Readded Chip8AllInOne.ch8

GamingMadster opened the pull request:
Still no clue why those hashes got all screwed.
Hopefully all is good now :D

GamingMadster committed:
maybe those SHA-1 Hashes won't screw themselves over again

GamingMadster commented:
...and apparently it says I changed 14k lines of the JSON. Neat.

Estus-Dev commented:
> ...and apparently it says I changed 14k lines of the JSON. Neat.

Well it looks like you added 11 and modified 7,282. They just show up as added and removed when you change them.

Without pulling it down to verify, it looks like it's line endings. Try setting your editor to use `\n` (Unix-style) endings rather than `\r\n` (Windows-style) and reformat the file. Also set git to convert `\r\n` line endings to `\n` when committing, most projects will expect that.

GamingMadster committed:
Line ending crap fixed

GamingMadster commented:
> > ...and apparently it says I changed 14k lines of the JSON. Neat.
> 
> Well it looks like you added 11 and modified 7,282. They just show up as added and removed when you change them.
> 
> Without pulling it down to verify, it looks like it's line endings. Try setting your editor to use `\n` (Unix-style) endings rather than `\r\n` (Windows-style) and reformat the file. Also set git to convert `\r\n` line endings to `\n` when committing, most projects will expect that.

Pretty sure I fixed it, or well, I hope so anyway.

Estus-Dev commented on database/programs.json:
```diff
@@ -3,53 +3,53 @@
     "description": "15 Puzzle, by Roger Ivie\n\nRearrange the tiles into the sequence 0-F. Moves tile by it associated key.",
     "release": "19xx",
-    "authors": ["Roger Ivie"],
+    "authors": [ "Roger Ivie" ],
```
Looks like Visual Studio is also formatting with spaces around strings. Can you close the file in visual studio and rerun `npm run format` please, just so there's not quite so much noise in the diff?

GamingMadster replied:
Just did so, now it's saying I added 8000+ lines.

¯\\_(ツ)_/¯

Estus-Dev replied:
Yeah. `/database/programs.json` is still wrong, but there's a new `/programs.json` that looks correct. Just move that into the `/database/` folder and you should be good.

GamingMadster replied:
Alright, I've done that. Should be all good to go.

GamingMadster committed:
todays episode on Does the Prettier Do Its Job...hopefully it did do its job.

GamingMadster committed:
Delete database/programs.json

GamingMadster committed:
Rename programs.json to database/programs.json

Estus-Dev committed:
fix: move entry to the bottom of the list

Estus-Dev committed:
Update hashes file for Chip8AllInOne by Madster

Timendus committed:
Lowercase SHA hash in programs.json

Timendus committed:
Lowercase SHA hash in sha1-hashes.json

Timendus commented:
Thanks for your contribution Madster! Hope the next one goes more smoothly :smile:
